addons/l10n_cl_website_sale/controllers/main.py

```diff
@@ -0,0 +1,50 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import _
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+from odoo.http import request
+
+
+class L10nCLWebsiteSale(WebsiteSale):
+
+    def _get_mandatory_fields_billing(self, country_id=False):
+        """Extend mandatory fields to add new identification and responsibility fields when company is chile"""
+        res = super()._get_mandatory_fields_billing(country_id)
+        if request.website.sudo().company_id.country_id.code == 'CL' and request.params.get('l10n_cl_sii_taxpayer_type', False) == '1':
+            res += ['vat']
+        return res
+
+    def values_postprocess(self, order, mode, values, errors, error_msg):
+        new_values, errors, error_msg = super().values_postprocess(order, mode, values, errors, error_msg)
+        if request.website.sudo().company_id.country_id.code == 'CL':
+            if mode[0] == 'new':
+                new_values['l10n_latam_identification_type_id'] = request.env.ref('l10n_cl.it_RUT').id
+            new_values['l10n_cl_sii_taxpayer_type'] = values.get('l10n_cl_sii_taxpayer_type', '')
+            if values.get('l10n_cl_activity_description'):
+                new_values['l10n_cl_activity_description'] = values.get('l10n_cl_activity_description')
+            if values.get('l10n_cl_dte_email'):
```

Comment:
l10n_cl_dte_email is apparently not in community, so we need either an extra module in enterprise or move everything to enterprise.